docs/getting-started/install/dreamhost.md

```diff
@@ -0,0 +1,163 @@
+# Install Dokku on DreamHost Cloud Server with cloud-init
+
+Cloud-init script can be used to automate installation of Dokku on
+Dreamhost (or any other OpenStack-compatible cloud with minimal
+changes).
+
+To create a new server and install Dokku from the openstack command
+line interface:
+
+```sh
+openstack server create \
+  --image Ubuntu-16.04 \
```

Comment:
Can we default this to 14.04 by any chance? Once we solve #2263, I can update this to default to 16.04.